### DISCUSSION TITLE
Lesson 6: Forge init not working correctly!!?

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#201] [Q&A] [answered]
So whenever we enter **forge init** in the foundry terminal, it gives us many files and folder like script, test, .github etc. But when I call it , it gives me everything except the **lib/forge-std**. What should I do?? (I use windows 10 by the way)

**Comment by rohit-bhalerao-7** [comment]
Try using `forge init --force` maybe that will work

**Comment by DevSwayam** [comment, marked as answer]
I came across same issue and i reffered this <a href="https://github.com/foundry-rs/foundry/issues/2207">Page<a> and i found a command to install lib folder seperately 

```terminal
git submodule add --force --name lib/forge-std https://github.com/foundry-rs/forge-std lib/forge-std
```
And for next project i re-installed forge using
```terminal
forge install
```
Now everything is working fine for me.

**Reply by 0xKumarShivam** [reply]
thanks Swayam

**Reply by DevSwayam** [reply]
Welcome

**Reply by satoshimasterbilly** [reply]
Hi Thanks so much I was wrestling with this same problem and very cool thanks for link-all good thanks Swayam

**Reply by DevSwayam** [reply]
Welcome 🤗

**Reply by Abdulllah-Rizwan** [reply]
Thank you soo much bro❤

**Reply by khushbusandhu** [reply]
thank you so much

**Reply by ShawnFromTexas** [reply]
Thanks! Fixed my issue as well! cheers!

**Reply by DevSwayam** [reply]
Glad to know it still lol

**Reply by Akashh-Dutta** [reply]
thanks.. it helped a lot

**Reply by DevSwayam** [reply]
Does this still works? Lol

**Reply by Akashh-Dutta** [reply]
yeah. and one more thing, 
I am totally new in this field. If you've seen patrick video, 1st part. He imported a file from lib folder named, script.sol... but I don't get any folder like that. Can you help me with this?

The script.sol supposed to deploy in anvil.

Thak you for reading.

**Reply by DevSwayam** [reply]
Can you raise a new discussion for this and tag me over there.

**Reply by Akashh-Dutta** [reply]
ok. thanks.. i will

**Comment by alymurtazamemon** [comment]
@DontMind-me Did you add the project name together?

```bash
forge init hello_foundry
```
replace it `hello_foundry` with whatever name you want

**Reply by Ifra001** [reply]
I'll try that right now!

**Comment by priyaccenture** [comment]
forge init is not working .

I got the below error 

```
error: 
The target directory is a part of or on its own an already initialized git repository,
and it requires clean working and staging areas, including no untracked files.

Check the current git repository's status with `git status`.
Then, you can track files with `git add ...` and then commit them with `git commit`,
ignore them in the `.gitignore` file, or run this command again with the `--no-commit` flag.

If none of the previous steps worked, please open an issue at:
https://github.com/foundry-rs/foundry/issues/new/choose
➜  foundry-simple-storage-f23 git:(master) ✗ 

```

**Reply by DevSwayam** [reply]
Did you delete your previous project setup?

**Reply by priyaccenture** [reply]
yes it worked .Thank you !!
